values.yaml

```
@@ -425,94 +425,129 @@ st2chatops:
 ## MongoDB HA configuration (3rd party chart dependency)
 ##
 ## For values.yaml reference:
-## https://github.com/helm/charts/tree/master/stable/mongodb-replicaset
+## https://github.com/bitnami/charts/tree/master/bitnami/etcd
 ##
 # Specs for the MongoDB image
-mongodb-ha:
+mongodb:
   # Change to `false` to disable in-cluster mongodb deployment.
   # Specify your external [database] connection parameters under st2.config
   enabled: true
   image:
     # StackStorm currently supports maximum MongoDB v4.0
     tag: "4.0"
+  ## MongoDB architecture. Allowed values: standalone or replicaset
+  ##
+  architecture: replicaset
+  ## Name of the replica set
+  ## Ignored when mongodb.architecture=standalone
+  ##
+  replicaSetName: rs0
+  ## Number of MongoDB replicas to deploy.
+  ## Ignored when mongodb.architecture=standalone
+  ##
+  replicaCount: 2
```

```suggestion
  replicaCount: 3
```
Any specific reason, why not `3` members for MongoDB cluster, as it was before?

---

This might not be intentional. As per the documentation, default value was 2. Though `values-prod.yaml` showed 4 which may not be necessary. I ran these changes with 3 and it was fine.

---

I've set this to 3